esp/src/eclwatch/HelpersWidget.js

```diff
@@ -284,7 +284,7 @@ define([
                 this.logData.push({
                     id: "H:" + i,
                     Type: helpers[i].Type,
-                    Description: helpers[i].IPAddress ? "//" + helpers[i].IPAddress + helpers[i].Name : helpers[i].Name,
+                    Description: helpers[i].Name,
```

Comment:
I suspect we would be better off with just pathtail(name)

Reply:
@kunalaswani this will translate to something like:
```
  function pathtail(path: string) {
      //  Assuming we need to support windows paths as well...
      const pathParts = path.split("\\").join("/").split("/"); 
      return pathParts.pop();
  }
```